statsmodels/tools/testing.py

```diff
@@ -16,8 +16,10 @@ def strip_rc(version):
 
 
 def is_pandas_min_version(min_version):
-    '''check whether pandas is at least min_version '''
-    return StrictVersion((pandas.__version__[:6])) >= min_version
+    '''check whether pandas is at least min_version
+    '''
+    from pandas import __version__ as pversion
+    return StrictVersion(strip_rc(pversion)) >= min_version
```

Comment:
This may work out of the box: https://github.com/statsmodels/statsmodels/blob/master/statsmodels/compat/scipy.py#L20

The only difference between ``pandas`` and ``numpy`` is the dev version scheme: ``1.2.3.dev0+githash`` vs. ``1.2.3+githash``.